src/deploy/missions/contracts/z-token/z-token.ts

```diff
@@ -86,35 +86,36 @@ IZNSContracts
   }
 
   getArtifact () {
-    return meowArtifact;
+    return zArtifact;
   }
 
+  // it will choose the governon as `admin` argument
+  // and deployAdmin as `minter` and first passed admin as `mintBeneficiary`.
```

Comment:
no. all these addresses should be parts of the config. we need to be able to specify these addresses as needed.